site/src/site/releasenotes/groovy-2.3.adoc

```diff
@@ -239,7 +239,7 @@ that you can choose from:
 
 * a simple strategy for creating chained setters
 * an external strategy where you annotate an explicit builder class
-while leaving some buildee class being built untouched
+while leaving some builder class being built untouched
```

Comment:
this is meant to be "buildee", perhaps it should be in italics to indicate it has a special meaning

Reply:
Changed to `_buildee class_`.